app/javascript/legacy_react/src/components/common/ProgressableButton.tsx

```diff
@@ -19,13 +19,13 @@ class ProgressableButton extends React.Component<ProgressableButtonProps, {}> {
   render() {
     let ourData: {title: string, disabled: boolean, prefix: JSX.Element|null}= {
       title:this.props.buttonText,
-      disabled:this.props.disabled,
+      disabled:this.props.disabled!,
```

Comment:
Are you sure `this.props.disabled` will always be non-null? If not, you can't make that assertion.